Watchman.DomainModel/Help/HelpInformation.cs

```diff
@@ -6,13 +6,15 @@ namespace Watchman.DomainModel.Help
 {
     public class HelpInformation : Entity
     {
+        public const ulong DEFAULT_SERVER_INDEX = 0;
+
         public Guid HelpId { get; set; }
         public IEnumerable<string> Names { get; set; }
         public string MethodFullName { get; set; }
         public IEnumerable<ArgumentInfo> ArgumentInfos { get; set; }
         public IEnumerable<Description> Descriptions { get; set; }
         public ulong ServerId { get; set; }
-        public bool IsDefault => ServerId == 0;
+        public bool IsDefault => ServerId == DEFAULT_SERVER_INDEX;
```

Comment:
myśle że 0 też będzie ok

Reply:
zrobiłem to, ponieważ w innym miejscu również odwołuję się do domyślnych helpów, a gdybyśmy kiedyś zmienili w tym miejscu, to byśmy mogli zapomnieć o tamtym
`var query = new GetHelpInformationQuery(HelpInformation.DEFAULT_SERVER_INDEX);`